src/generate.js

```diff
@@ -17,11 +17,15 @@ self.generate = function(options) {
 	if (!options.hasOwnProperty('uppercase')) options.uppercase = true;
 	if (!options.hasOwnProperty('excludeSimilarCharacters')) options.excludeSimilarCharacters = false;
 
+	// check if must limit the number of repeating characters
+	if (!options.hasOwnProperty('repeatCharactersLimit') || typeof options.repeatCharactersLimit !== 'number' || options.repeatCharactersLimit < 2) options.repeatCharactersLimit = undefined;
```

Comment:
Sorry if I was unclear, the `typeof options.xxx !== 'number'` should be unnecessary :) type checking might be a good idea but there is no precedence to do it already. Perhaps another PR?